src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -1718,9 +1718,10 @@ function dhcpd_dhcrelay4_configure($verbose = false)
         }
 
         if (!isset($destif)) {
-            if (is_array($config['gateways']['gateway_item'])) {
-                foreach ($config['gateways']['gateway_item'] as $gateway) {
-                    if (isset($gateway['defaultgw'])) {
+            $gateways = iterator_to_array((new Gateway)->gatewayIterator());
```

Comment:
that one is still missing the path btw

Reply:
woops, seems i missed the last grep entry